src/emailpro/account/update/emailpro-account-update.html

```diff
@@ -112,7 +112,7 @@
                      data-ng-class="{ 'has-error': selectedAccount.passwordConfirmation && differentPasswordFlag }">
                     <label for="passwordConfirmation" class="col-md-3 control-label"
                            data-ng-class="{ 'required': !selectedAccount.canBeConfigured }"
-                           data-translate="emailpro_ACTION_update_account_step1_password_confirmation_label"></label>
+                           data-translate="exchange_account_add_passwordConfirmation_label"></label>
```

Comment:
I'm curious about the fact that you are here replacing the translation key with this new prefix `exchange_` whereas we are in the `emailpro` module.

Reply:
It was cross referenced in manager-web i think. 
Have fixed it now. It was a problem of inconsistent translations across EN_GB and other English files. Dont know if i should update this or if this should go through translation team though!